simple-serial-port/simple-serial-port/SimpleSerial.cpp

```diff
@@ -119,7 +75,7 @@ void SimpleSerial::init(const std::string &com_port, DWORD COM_BAUD_RATE)
     }
 }
 
-void SimpleSerial::CustomSyntax(const std::string &syntax_type)
```

Comment:
It could be my mistake, but nor Microsoft compiler nor G++ were able to build SimpleSerial because of mismatching references in the header files and in the source.
Back to the class definition we have: 
> void CustomSyntax(std::string syntax_type);  
  
But in the function definition:
  
> void SimpleSerial::CustomSyntax(const std::string &syntax_type) { ... }
  
Causing compilers not to found the functions references.

Reply:
replacing `const string&` with `string` fixed it